modules/manage/partials/tiered-storage.adoc

```diff
@@ -575,9 +576,24 @@ Microsoft ABS/ADLS::
 +
 --
 
-
 include::manage:partial$azure-blob-limitations.adoc[]
 
+To configure access to an Azure container with a managed identity:
+
+. Configure an xref:manage:security/iam-roles.adoc#configuring-iam-roles[Azure managed identity].
+. Run the `rpk cluster config edit` command, then edit the following required properties:
++
+[,properties]
+----
+cloud_storage_enabled: true
+cloud_storage_credentials_source: azure_vm_instance_metadata
```

Comment:
@andijcr @c4milo Does the config value need to be enclosed in quotes?

Reply:
no